platform/view/services/view/mock/runner.go

```diff
@@ -25,61 +26,61 @@ type Responders struct {
 	Channel       *session.LocalBidirectionalChannel
 }
 
-type MockContext struct {
+type DelegatedContext struct {
```

Comment:
I am wondering if we can move the `DelegatedContext` into the integration test where it is exclusively used, and even make it private.